### PR TITLE
add mongodb + digest auth

### PR DESCRIPTION
**Pull request by ozhey**

### CODE REVIEW

File: plugins/connections/digest_auth.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	"fmt"
+	"github.com/blinkops/blink-http/consts"
+	log "github.com/sirupsen/logrus"
+	"io"
+	"net/http"
+	"strings"
+)
+
+func DigestAuth(req *http.Request, conn map[string]string, baseUrl string) error {
+	dummyReq, err := http.NewRequest(http.MethodGet, req.URL.String(), nil)
```
Comment (blink-daniel) [inline plugins/connections/digest_auth.go:16]: please add err check

File: plugins/connections/digest_auth.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	client := &http.Client{}
+	resp, err := client.Do(dummyReq)
+	if err != nil {
+		return err
```
Comment (blink-daniel) [inline plugins/connections/digest_auth.go:20]: please wrap error

**Comment by blink-daniel** [comment]
+ please add you connection type to plugin.yaml

File: plugins/mongodb-atlas/named_actions.yaml [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+          List of objects that provides the pairings of one role with one applicable database. For example:
+          [
+            {
+              "collectionName": "Recipes",
+              "databaseName": "recipes",
+              "roleName": "read"
+            }
+          ]
+        required: true
+        index: 3
```
Comment (DanielDinsh) [inline plugins/mongodb-atlas/named_actions.yaml:116]:
```suggestion
        index: 2
```

File: plugins/mongodb-atlas/named_actions.yaml [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+      - name: username
+        display_name: Username
+        description: The username of the database user to be updated.
+        type: text
+        required: true
+        index: 4
```
Comment (DanielDinsh) [inline plugins/mongodb-atlas/named_actions.yaml:122]:
```suggestion
        index: 3
```

File: plugins/connections/digest_auth.go (hunk 1 above)
Comment (ozhey) [reply plugins/connections/digest_auth.go:16]: oops

**Review by blink-daniel — APPROVED** [review]

**Pull request merged**